### ISSUE TITLE
Spelling mistakes in trial.xhtml howto

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6406]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#6406|
|Type|defect|
|Created|2013-04-06 00:37:34Z|
|Branch|https://github.com/twisted/twisted/tree/trial-howto-spelling-6406|

`doc/core/howto/trial.xhtml` has a couple of spelling mistakes, at least: "particulary" and "mutiply".

Attachments:

* [twisted-doc-core-howto-trial.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/471/4711c30380bc740314ed56097853176e46198aef/f36bbe1d08d68312bd1d83c7b7332b2d730ebb74.patch) (1388 bytes) - added by nothung on 2013-04-20 19:20:06Z - Corrected misspellings in doc/core/howto/trial.xhtml for words "multiply", "particularly", and "possible"
* [twisted-doc-core-howto-trial_fixed.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/471/4711c30380bc740314ed56097853176e46198aef/82e55e8aeb95ac744c95dbafa9b3e46bd40a3647.patch) (11108 bytes) - added by phoenix on 2013-09-23 13:10:10Z - Corrected issues mentioned by RichardW

<details><summary>Searchable metadata</summary>

```
trac-id__6406 6406
type__defect defect
reporter__itamar itamar
priority__normal normal
milestone__ 
branch__branches_trial_howto_spelling_6406 branches/trial-howto-spelling-6406
branch_author__rwall rwall
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation_gsoc_easy documentation gsoc easy
time__1365208654000000 1365208654000000
changetime__1379948342000000 1379948342000000
version__None None
owner__rwall rwall

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="nothung's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|nothung set owner to nothung<br>nothung set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="nothung's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|nothung removed owner<br>nothung set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_3"></a>|@wallrj set owner to @wallrj<br>@wallrj set status to `assigned`|
|-|-|

reviewing...

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_4"></a>|@wallrj set owner to nothung<br>@wallrj set status to `new`|
|-|-|

Code Review:

Thanks nothung for correcting those spelling mistakes. It looks like
you found them all.

Please address or answer the following points:
 1. Add a .doc news file. All patches should include a news file. This
    is a documentation change so it should have a .doc extension and a
    short description of the change.

    https://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles

 2. There are a few sentences in that document that I either don't
    make sense or sound unprofessional or which might be difficult for
    non-english speakers to interpret.

    Read through the document and fix at least the grammar and
    punctuation errors.

    Raise another ticket for general improvements to the trial howto
    document.

    Here are some things I noticed as I read the document. Fix at
    least the ones marked with an X

    1. "There is also a ton of information"
    2. "cover a networking code" X
    3. "get set up"
    4. "get these both done"
    5. "you're in business"
    6. "(though your files are probably not in /tmp" - missing closing
       bracket X
    7. "How to interpret this output?"
    8. "Trial create a _" X
    9. "is a copy of test_base_1" X
    10. "used in tests methods" X
    11. "Much cleaner, no?"
    12. "If you run the above, you'll see"
    13. "let that exception go by as TypeError is already" X
    14. "We'll now create a custom protocol to invoke our class from within a telnet-like session"
    15. "it helps a lot to be comfortable with the"
    16. "The server (once again, see below) hands the work off to" X
    17. "If you run this test (trial calculus.test.test_remote_1), everything should be fine"
    18. "Of course, what we build is not particulary useful for now :
        we'll now build a client to our server" X
    19. "Thus, every callLater calls in the protocol will finish" X
    20. "that simulates and advance in time" X
    21. "when everything went fine"
    22. "people that want" X
    23. "avoid that like the plague"
    24. "Recent versions of trial will fail loudly if you remove the stopListening call, which is good."
    25. "tearDown will called in any case" X
    26. "expect that every objects you" X
    27. "This remove the need" X
    28. "Generally an user" X
    29. "I have this strange problem on my crappy network environment"
    30. "There are different kind of measures" X
    31. "for every modules used" X
    32. "the number of times you went through during the tests" X

Once you've addressed those points, please submit another patch for
review.

-RichardW.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

> Add a .doc news file. All patches should include a news file. This is a documentation change so it should have a .doc extension and a short description of the change.

I think typo fixes can be a .misc file.

**Comment by twisted-trac** [comment]
|<img alt="phoenix's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_7"></a>|phoenix commented|
|-|-|

Corrected issues mentioned by RichardW.

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_8"></a>|@wallrj commented|
|-|-|

(In [40060]) Branching to 'trial-howto-spelling-6406'

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_9"></a>|@wallrj commented|
|-|-|

(In [40062]) Apply  twisted-doc-core-howto-trial_fixed.patch from phoenix. Refs [#6406](https://github.com/twisted/twisted/issues/6406)

**Comment by twisted-trac** [comment]
|<img alt="phoenix's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|phoenix removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_11"></a>|@wallrj set owner to @wallrj<br>@wallrj set status to `assigned`|
|-|-|

Replying to [phoenix](#note_7):
> Corrected issues mentioned by RichardW.

Thanks pheonix, Chris, and holger,

The build results look good and I agree with almost all your changes.

I changed a couple of things in r40064.

Added a news file r40065.

So I'll merge it.

Thanks.

-RichardW.

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_12"></a>|@wallrj set status to `closed`|
|-|-|

(In [40067]) Merge trial-howto-spelling-6406

Authors: nothung,phoenix,PyWebEU
Reviewer: rwall
Fixes: [#6406](https://github.com/twisted/twisted/issues/6406)

Spelling and grammar corrections in the trial howto document.
